apps/root/src/hooks/useNetWorth.ts

```diff
@@ -95,8 +98,35 @@ const useNetWorth = ({ walletSelector, chainId }: NetWorthProps) => {
     0
   );
 
+  // EARN
+  let earnAssetsTotalValue = 0;
+  let filteredUserStrategies = userStrategies;
+  if (walletSelector !== ALL_WALLETS) {
+    if (chainId && walletAddressToEvaluate) {
```

Comment:
This condition can be simplified with the condition above right? And maybe in DCA too 😄

Reply:
I mean yeah but we still ahve that other condition and it is in the scope of the `1walletSelector !== ALL_WALLETS`